Content.Tests/DMProject/Tests/List/alist/AListAdd.dm

```diff
@@ -3,7 +3,9 @@
 	AL.Add("c", "d")
 	ASSERT(AL["c"] == -4)
 	ASSERT(AL["d"] == null)
+	ASSERT(AL.len == 4)
 
 	AL.Add(list("c", "d"))
 	ASSERT(AL["c"] == -4)
-	ASSERT(AL["d"] == null)
\ No newline at end of file
+	ASSERT(AL["d"] == null)
+	ASSERT(AL.len == 4)
```

Comment:
This is failing because `DreamAssocList` doesn't override `TryGetVar()` to return a value for the `len` var.